posthog/heatmaps/heatmaps_api.py

```diff
@@ -252,7 +252,10 @@ def _return_heatmap_coordinates_response(query_response: HogQLQueryResponse) ->
 
         response_serializer = HeatmapsResponseSerializer(data={"results": data})
         response_serializer.is_valid(raise_exception=True)
-        return response.Response(response_serializer.data, status=status.HTTP_200_OK)
+
+        resp = response.Response(response_serializer.data, status=status.HTTP_200_OK)
+        resp["Cache-Control"] = "max-age=30"
```

Comment:
if you're clicking around a lot (as i was testing this) there's no point running this query over and over